app/models/trainer.rb

```diff
@@ -1,3 +1,4 @@
 class Trainer < ApplicationRecord
     has_many :pokemons
+    has_many :teams
```

Comment:
This should be `has_one` because a Trainer should only have one team (as far as I know lol).